_docs/Getting-Started.md

```diff
@@ -59,17 +57,31 @@ You can also change the styling of the component by changing the `src\stylesheet
 
 Once you have modified your component and are ready to test it, you need to run the build command, so the code can be built.
 
-To do so, run the following command, replacing `TestComponent` with the name of your component:
+To do so, run the following command, provided by the Makefile generated with the project:
```

Comment:
I would change it to:
To do so, run the following command, provided by the Makefile generated with the project. Remember to replace `TestComponent` with the name of your component.